app/services/remotestorage.js

```diff
@@ -1,58 +1,78 @@
-import Ember from 'ember';
+import Service from '@ember/service';
+import RemoteStorage from 'npm:remotestoragejs';
+import Widget from 'npm:remotestorage-widget';
+import Shares from 'npm:remotestorage-module-shares';
 
-export default Ember.Service.extend({
+export default Service.extend({
 
+  rs: null, // remoteStorage instance
+  shares: null,
+  widget: null,
   connected: false,
   connecting: false,
 
   setup: function() {
-    RemoteStorage.WireClient.REQUEST_TIMEOUT = 90000;
+    let remoteStorage = new RemoteStorage({
+      cache: false,
+      requestTimeout: 90000,
+      modules: [Shares.default]
+    });
 
     remoteStorage.access.claim('shares', 'rw');
-    remoteStorage.displayWidget({ redirectUri: window.location.href });
 
     remoteStorage.on('ready', () => {
-      console.log('RS ready');
+      console.debug('RS ready');
       this.setProperties({
         'connecting': false,
         'connected': true
       });
     });
     remoteStorage.on('not-connected', () => {
-      console.log('RS not-connected');
+      console.debug('RS not-connected');
       this.setProperties({
         'connecting': false,
         'connected': false
       });
     });
     remoteStorage.on('connected', () => {
-      console.log('RS connected');
+      console.debug('RS connected');
       this.setProperties({
         'connecting': false,
         'connected': true
       });
     });
     remoteStorage.on('disconnected', () => {
-      console.log('RS disconnected');
+      console.debug('RS disconnected');
       this.setProperties({
         'connecting': false,
         'connected': false
       });
     });
     remoteStorage.on('connecting', () => {
-      console.log('RS connecting');
+      console.debug('RS connecting');
       this.setProperties({
         'connecting': true,
         'connected': false
       });
     });
     remoteStorage.on('authing', () => {
-      console.log('RS authing');
+      console.debug('RS authing');
       this.setProperties({
         'connecting': true,
         'connected': false
       });
     });
+
+    this.set('rs', remoteStorage);
+    this.set('shares', remoteStorage.shares);
+
+    let widget = new Widget(remoteStorage, {
+      redirectUri: window.location.href
```

Comment:
`redirectUri` is not a valid option (yet).

Reply:
Thanks. Not sure where it came from. Probably the old widget/RS.